types/parse/index.d.ts

```diff
@@ -621,15 +658,16 @@ subscription.on('close', () => {});
      */
     class User extends Object {
 
+        static allowCustomUserClass(isAllowed: boolean): void;
+        static become(sessionToken: string, options?: SuccessFailureOptions): Promise<User>;
```

Comment:
`SuccessFailureOptions` is incorrect here, see the list of accepted options [here](https://github.com/parse-community/Parse-SDK-JS/blob/master/src/ParseUser.js#L637)